cms/templates/settings.html

```diff
@@ -333,6 +333,9 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <li class="field text" id="field-course-overview">
                   <label for="course-overview">${_("Course Overview")}</label>
                   <textarea class="tinymce text-editor" id="course-overview"></textarea>
+                  <label class="sr" for="cm-textarea">
+                    Code Editor
```

Comment:
All strings exposed to the user (even screenreader strings) must be internationalized.